playbooks/roles/ocp-cli-plugins/tasks/install-cli-plugin-required-packages.yml

```
@@ -0,0 +1,15 @@
+---
+
+## OCP CLI-plug-in required package installation module
+
+- name: Install prerequisite packages
+  yum:
+    name: "@Development tools"
+    state: latest
+
+- name: Install golang with version>=1.18
+  include_role:
+    name: golang-installation
+  vars:
+    golang_tarball_url: "{{ golang_tar_url | default('https://go.dev/dl/go1.18.4.linux-ppc64le.tar.gz', true) }}"
```

Please use `go_tarball`
```suggestion
    go_tarball: "{{ golang_tar_url | default('https://go.dev/dl/go1.18.4.linux-ppc64le.tar.gz', true) }}"
```